### PR TITLE
ReplacedPromise.allSettled by RSVP.allSettled

### PR DESCRIPTION
**Pull request by mohak852**
Fixes #
I have replaced ReplacedPromise by RSVP>allSettled
 
YOu can go through the changes and can merge this into your original project

### CODE REVIEW

File: app/controllers/public/index.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ export default class IndexController extends Controller {
         .catch(async e => {
           console.error('Error while saving order', e);
           try {
-            await Promise.allSettled((attendees ? attendees.toArray() : []).map(attendee => attendee.destroyRecord()));
+            await RSVP.allSettled((attendees ? attendees.toArray() : []).map(attendee => attendee.destroyRecord()));
```
Comment (niranjan94) [inline app/controllers/public/index.js:139]: Codacy Issue found: ['RSVP' is not defined.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5581106)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5581106&bid=18294771)

File: app/controllers/public/index.js (hunk 1 above)
Comment (mohak852) [reply app/controllers/public/index.js:139]: Sir what I can do for it any Suggestions?

**Comment by prateekj117** [comment]
@mohak852 What issue does this fix? Please fill the PR template accordingly with more details.

**Comment by iamareebjamal** [comment]
As you can already see, this is not correct and the build is failing. You haven't even imported the function you are replacing it with

**Comment by mariobehling** [comment]
This PR has been stale for a long time. Therefore closing it.

**Pull request closed without merging**